.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,123 @@
+<component name="ProjectCodeStyleConfiguration">
+  <code_scheme name="Project" version="173">
+    <JetCodeStyleSettings>
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </JetCodeStyleSettings>
+    <codeStyleSettings language="XML">
```

Comment:
toto nejde specifikovat v tom editorconfig?

Reply:
kouknu. Pamatuju si to tak, že sice to specifikovat lze, ale stejně si to IDE vytvoří samo (navíc).

Reply:
v tomhle PR to smáznu, přesouvám dotaz do https://github.com/alesrazym/rocket-app/pull/8

A rebasnu to na aktuální develop